switch-configuration/config/vlans

```diff
@@ -1,9 +1,11 @@
 // Expo Center -- VLANS 100-499
-#include vlans.d/Expo
+//#include vlans.d/Expo
 
 // Conference Center -- VLANS 500-899
-#include vlans.d/Conference
+//#include vlans.d/Conference
 
 // Bridged -- VLANS 900-999 (carried across fiber between buildings)
-#include vlans.d/Bridged
+//#include vlans.d/Bridged
 
+// Hilton -- SCaLE 19x special
```

Comment:
With `vlans.d` does this mean that we actually have a holistic view for both pasadena and lax hilton? I know we wont be at the hilton past this year but it does seem promising that we have the ability to include additional venues if/when the time comes via this approach. And all you have to do is comment out which of the `includes` you dont want in the future.

Reply:
Yes... vlans.d essentially contains files included by directives in the vlans file in the parent directory. By turning those on and off, we can include or not include various files in the vlans.d directory. However, currently, we couldn't include both because the Hilton VLANs and the CF VLANs would collide.

Reply:
totally makes sense